Sample Applications/DataBindingDemo/App.cs

```diff
@@ -92,5 +96,62 @@ private void LoadAuctionData()
 
             #endregion
         }
+
+        private void OnGridFocus(object sender, RoutedEventArgs e)
+        {
+            var sen = sender as Grid;
```

Comment:
All of this seems like it could be done in XAML, let's chat tomorrow.

Reply:
Thanks for the help. I have made the changes